assets/sass/local.sass

```diff
@@ -1,2 +1,5 @@
 .govuk-main-wrapper
   min-height: 600px
+
+.moj-identity-bar__title
+  font-size: 1.1875rem
```

Comment:
You can use [`govuk-font`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-font) here